src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1048,6 +1048,12 @@ public string[] LiteralPath
         [Parameter]
         public SwitchParameter List { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating if the search contains the specified characters.
```

Comment:
This is a bit unclear, as Select-String already returns data containing the specified characters. Does this parameter affect the _search_ or the _result_ that the user sees or receives?

Reply:
Sorry, the parameter should be affecting the result that the user sees. I will update the comment and re-commit.